addOns/client/src/main/java/org/zaproxy/addon/client/ExtensionClientIntegration.java

```diff
@@ -146,15 +157,17 @@ public void initModel(Model model) {
                                 new ClientSideDetails(
                                         Constant.messages.getString("client.tree.title"), null),
                                 this.getModel().getSession()));
+        spiderScanController = new SpiderScanController(this);
+        passiveScanController = new ClientPassiveScanController();
     }
 
     @Override
     public void hook(ExtensionHook extensionHook) {
         super.hook(extensionHook);
 
-        scanController = new ClientPassiveScanController();
         this.api = new ClientIntegrationAPI(this);
```

Comment:
`ClientIntegrationApi`

Reply:
Thats existing code.

Reply:
Shouldn't matter to anyone but us.

Reply:
Agreed, but thats not a change I introduced in this PR. Happy for anyone to rename it in another PR, but in general I dont think we should ask people to fix existing code in PRs unless its directly related to the changes being made.